crates/biome_html_analyze/tests/specs/a11y/noNoninteractiveElementInteractions/html/invalidResourceEvents.html

```
@@ -0,0 +1,4 @@
+<!-- should generate diagnostics -->
+<img src="image.png" alt="Example" onerror="handleError()" onclick="handleInteraction()" />
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)